Edits Account Generation Page (#5042)
w3f/polkadot-wiki: docs/learn/learn-account-generation.md

## Ways to Generate an Account

:::info Treasury-funded Wallets and Browser Extensions

> You can create accounts using treasury-funded
> [wallets and extensions](../general/wallets-and-extensions.md). Bear in mind that some of these are
> **unaudited** and are not officially affiliated with Web3 Foundation or the
> {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} project unless otherwise stated.

Reviewer: ```suggestion
If you are a general user, you are recommended to create accounts using any of the
[wallets and extensions](../general/wallets-and-extensions.md) funded by the Treasury or Grants. If you are a power user or a developer, you can use [Polkadot-JS Extension](../polkadotjs.md#polkadot-js-extension).
```

:::

> You can also create accounts using
> [the Polkadot-JS browser extension](#polkadot-js-browser-extension). This extension remembers your

Reviewer: ```suggestion
When you create accounts using browser extensions like
[the Polkadot-JS extension](#polkadot-js-browser-extension), it stores your
```

> accounts and lets you clear your browser cache without fear. Remember to back up your seed phrase -

Reviewer: ```suggestion
accounts in a private vault and lets you clear your browser cache without fear. Remember to back up your seed phrase -
```

if you lose access to this computer or the extension somehow crashes beyond repair, the seed phrase
will come in handy. Alternatively, you can create accounts using the
[Polkadot-JS UI](#polkadot-js-ui), although this option is disabled by default. More info about the
Polkadot-JS UI can be found on [the dedicated page](../general/polkadotjs-ui.md).

> Please note that as this keeps your accounts in the browser extension, holding significant funds on
> them is unsafe. By definition, a browser is a "hot wallet" susceptible to a wide range of attacks,
> so keep your funds in cold storage when dealing with non-trivial amounts.

Reviewer: ```suggestion
Please note that all wallets that are connected to the internet are considered "hot wallets" that are susceptible to a wide range of attacks,
so it is recommended to use cold storage when dealing with non-trivial amounts.
```

Current popular cold storage solutions within the
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} ecosystem include
[Ledger](../general/ledger.md) and [Polkadot Vault](../general/polkadot-vault.md). The latter keeps

> your keys on an air-gapped mobile phone. However, it does require obtaining an old Android or
> iOS-compatible phone that you are comfortable using only for Polkadot Vault.

Reviewer: ```suggestion
your keys on an air-gapped mobile phone. However, it does require obtaining a dedicated Android or
iOS-compatible phone that you are comfortable using only for Polkadot Vault.
```
